lib/plottr_firebase/migrations/mark-deleted-authorised-documents.js

```diff
@@ -60,26 +142,32 @@ const deletedFiles = (userId) => {
             if (document.exists) {
               return {
                 document: document.data(),
-                exists: true
+                exists: true,
               }
             }
             return {
-              exists: false
+              exists: false,
             }
           })
-      }))
-        .then((results) => {
-          return results
-            .filter(({ exists, document }) => {
-              return exists && document.deleted
-            })
-            .map(({ document }) => document)
+      })
+    ).then((results) => {
+      return results
+        .filter(({ exists, document }) => {
+          return exists && document.deleted
         })
+        .map(({ document }) => document)
     })
+  })
 }
 
-const markDeletedAuthorisedDocumentsAsDeleted = (userId) => {
+const markDeletedAuthorisedDocumentsAsDeleted = (userId, executingUserId, readOnly) => {
   const database = admin.firestore()
+  const runTransactionWithAuditing = logChange(readOnly)(
+    database,
+    executingUserId,
+    'copy-known-file-information-to-authorisations.js',
```

Comment:
i think this line should be something to do with this migration, right? Something along the lines of 'mark-deleted-authorised-documents-as-deleted`

Reply:
Same thing here... thanks for spotting these! :)